src/pixelator/statistics.py

```diff
@@ -131,6 +131,22 @@ def log1p_transformation(df: pd.DataFrame) -> pd.DataFrame:
     return log1p_df
 
 
+def rate_diff_transformation(df: pd.DataFrame) -> pd.DataFrame:
+    """Transform antibody counts based on deviation from an expected baseline distribution.
+
+    In this function we refer to baseline distribution as fixed ratio of different antibody types in each node. For example, if in total 10% of antibodies are HLA-ABC, in a node with 120 antibodies we expect to see 12 HLA-ABC counts. If we actually see 8 counts in this node, the rate_diff_transformation for HLA-ABC in this node will be -4.
```

Comment:
We should break this line up into lines that are roughly 80 characters long. Unfortunately ruff in unable to autoformat this.